Common subdirectories: grpcio-tools-1.59.3.old/grpcio_tools.egg-info and grpcio-tools-1.59.3/grpcio_tools.egg-info
Common subdirectories: grpcio-tools-1.59.3.old/grpc_root and grpcio-tools-1.59.3/grpc_root
Common subdirectories: grpcio-tools-1.59.3.old/grpc_tools and grpcio-tools-1.59.3/grpc_tools
diff -u grpcio-tools-1.59.3.old/setup.py grpcio-tools-1.59.3/setup.py
--- grpcio-tools-1.59.3.old/setup.py	2023-11-26 17:17:20.186735538 +0800
+++ grpcio-tools-1.59.3/setup.py	2023-11-26 17:17:44.826734253 +0800
@@ -90,7 +90,7 @@
     )
     cxx = os.environ.get("CXX", "c++")
     cpp_test = subprocess.Popen(
-        [cxx, "-x", "c++", "-std=c++14", "-"],
+        [cxx, "-x", "c++", "-std=c++17", "-"],
         stdin=PIPE,
         stdout=PIPE,
         stderr=PIPE,
@@ -101,7 +101,7 @@
     # Double-check to see if -latomic actually can solve the problem.
     # https://github.com/grpc/grpc/issues/22491
     cpp_test = subprocess.Popen(
-        [cxx, "-x", "c++", "-std=c++14", "-", "-latomic"],
+        [cxx, "-x", "c++", "-std=c++17", "-", "-latomic"],
         stdin=PIPE,
         stdout=PIPE,
         stderr=PIPE,
@@ -138,7 +138,7 @@
 EXTRA_ENV_COMPILE_ARGS = os.environ.get("GRPC_PYTHON_CFLAGS", None)
 EXTRA_ENV_LINK_ARGS = os.environ.get("GRPC_PYTHON_LDFLAGS", None)
 if EXTRA_ENV_COMPILE_ARGS is None:
-    EXTRA_ENV_COMPILE_ARGS = "-std=c++14"
+    EXTRA_ENV_COMPILE_ARGS = "-std=c++17"
     if "win32" in sys.platform:
         if sys.version_info < (3, 5):
             # We use define flags here and don't directly add to DEFINE_MACROS below to
Common subdirectories: grpcio-tools-1.59.3.old/third_party and grpcio-tools-1.59.3/third_party
